packages/x-charts/README.md

```diff
@@ -1,5 +1,33 @@
 # @mui/x-charts
 
-Package under development.
+This package is the community edition of the charts components.
+It's part of MUI X, an open-core extension of MUI, with advanced components.
 
-The alpha phase will start in the coming months, stay tuned!
+## Installation
+
+Install the package in your project directory with:
+
+```sh
+npm install @mui/x-charts
+```
+
+or
+
+```sh
+yarn add @mui/x-charts
+```
+
+This component has the following peer dependencies that you will need to install as well.
+
+```json
+"peerDependencies": {
+    "@mui/material": "^5.4.1",
+    "@mui/system": "^5.4.1",
+    "react": "^17.0.0 || ^18.0.0",
+    "react-dom": "^17.0.0 || ^18.0.0"
+},
+```
+
+## Documentation
+
+[The documentation](https://mui.com/x/react-charts)
```

Comment:
Fixed in https://github.com/mui/mui-x/commit/db1a54b5626c68db56ddf848a97e5f21c3427b39